craft/pio/src/pio.scala

```diff
@@ -36,9 +36,13 @@ class Lpio(c: pioParams)(implicit p: Parameters) extends LpioBase(c)(p)
 {
 
 // User code here
+  override def extraResources(resources: ResourceBindings) =
+      Map("pio-width" -> Seq(ResourceInt(pioWidth)),
+          "pio-type" -> Seq(ResourceString("loopback")))
```

Comment:
This "pio-type" isn't really a property of the PIO itself. That's describing the fact that we've put a loopback on it in our test harness.

Reply:
I was trying to come up with a fake example to show how it could be used. If that is ok, I will merge

Reply:
I'm afraid that this example may cause more confusion than it provides help, since it may cause people to think that this is actually linked somehow to the other hardware block in this repo with the name "loopback".

I'd rather leave out fake examples, and instead come up with real examples. This is a perfectly usable, real IP block, albeit simple, and there's nothing preventing someone from actually using this block in a real design, so I don't think we should pollute it with fake examples. If we want to provide other examples of resource types, I think we can include it in documentation we have on this (and how the Resources system works in general), but I don't think we necessarily need to include them all here. There are examples of other Resource types throughout rocket-chip.

I would strongly request that we remove this outright, or that we at least replace it with some other property that actually describes this block.